player.go

```diff
@@ -140,3 +140,55 @@ func (a *API) Queue(uri string) error {
 
 	return a.post("v1", "/me/player/queue", v, nil)
 }
+
+// PlayPlaylist sets the playback on a playlist context and not a single song uri.
+// https://community.spotify.com/t5/Spotify-for-Developers/API-is-there-a-way-to-start-playing-a-playlist-from-a-specific/m-p/4977046#M509
+// https://stackoverflow.com/questions/48532890/spotify-web-api-play-endpoint-play-track
+func (a *API) PlayPlaylist(deviceID, playlistURI string) error {
```

Comment:
This function and the next (`PlayAlbum()`) appear to be exactly the same as `Play()`. Why don't we delete them and simply use `Play()` for all three scenarios?

Reply:
Play uses a 'uri' body patan but for albums and playlists you need to use a 'context_uri'.